config/application.rb

```diff
@@ -1,21 +1,29 @@
-require_relative 'boot'
+require_relative "boot"
 
 require "rails"
+# Pick the frameworks you want:
 require "active_model/railtie"
+require "active_job/railtie"
 require "active_record/railtie"
+# require "active_storage/engine"
 require "action_controller/railtie"
+# require "action_mailer/railtie"
+# require "action_mailbox/engine"
+# require "action_text/engine"
 require "action_view/railtie"
+# require "action_cable/engine"
```

Comment:
All these lines commented out... what do you think about deleting them? Personally prefer a clean file without "just in case"s. If this application ever needs Action Text, then we can add the `require` call.

What do you think?

Reply:
I find that make harder to upgrade Rails since every time we run the `app:update` task we will see a bigger diff than it is necessary. But that is up to you.

Reply:
Yeah, if you don't feel srong about that, I prefer to optimmize for reading the source code. Prefer a minimal and clean file.

Reply:
Removed the comments.